### DISCUSSION TITLE
Modifiers

### DISCUSSION THREAD

**Discussion by JaworB** [mbround18/valheim-docker#792] [Q&A] [answered]
Hi,
I've got question regarding modifiers syntax in compose file.
If I want to use multiple modifiers should it look like that? 

`MODIFIERS: portals=casual,raids=none`
or
`MODIFIERS: "portals=casual,raids=none"`

I'm not sure cause in both examples server browser in Valheim says that there is no modifiers on my server

**Comment by mbround18** [comment, marked as answer]
Either quoted or unquoted works, can you exec into the container and check the config file that exists in the game root?

**Reply by JaworB** [reply]
Thanks for confirmation.
Yes, I've found these lines:

`  "modifiers": [
    {
      "name": "portals",
      "value": "casual"
    },
    {
      "name": "raids",
      "value": "none"
    }`
    
    And looks like everything works as intended.

**Comment by an-englishman** [comment]
Hey, Im running your Valheim docker container on my synology. Could you let me know where this file is found? Can I also just add my modifiers to this file and it will update the world? Thanks for all the hard work.

**Reply by mbround18** [reply]
In the config.json at the root of the game server try running in debug mode to see if its applying the modifiers
